generic_server.cc

```diff
@@ -228,7 +237,17 @@ future<> server::do_accepts(int which, bool keepalive, socket_address server_add
             auto addr = std::move(cs_sa.remote_address);
             fd.set_nodelay(true);
             fd.set_keepalive(keepalive);
-            auto conn = make_connection(server_addr, std::move(fd), std::move(addr));
+            auto conn = make_connection(server_addr, std::move(fd), std::move(addr),
+                    seastar::gate::holder(_gate_uninitialized_conns));
+
+            if (size_t c =_gate_uninitialized_conns.get_count();
+                    c > _max_uninitialized_connections) {
```

Comment:
We need a way to disable this connection shedding. A typical way to do this is to treat either `0` or `std::numeric_limits<uint32_t>::max()` as the sentinel value which disables the feature.

Reply:
done